web/src/compose_banner.ts

```diff
@@ -23,6 +23,7 @@ const MESSAGE_SENT_CLASSNAMES = {
     sent_scroll_to_view: "sent_scroll_to_view",
     narrow_to_recipient: "narrow_to_recipient",
     message_scheduled_success_compose_banner: "message_scheduled_success_compose_banner",
+    guest_in_private_recipient: "guest_in_private_recipient",
```

Comment:
`guest_in_direct_message_recipient`; `private messages` got renamed to `direct messages`, and there's no reason to introduce new things using the old terminology.

Reply:
Sure! I have to work on naming things properly 😅